src/west/manifest.py

```diff
@@ -849,12 +853,17 @@ def posixpath(self) -> Optional[str]:
     def name_and_path(self) -> str:
         return f'{self.name} ({self.path})'
 
+    @property
+    def description(self) -> Optional[str]:
+        return self.description
```

Comment:
Doesn't this cause infinite recursion? I think it is not necessary to have the property.

Reply:
Yep, you are right. Leftover from a first pass where there was a `_description` field.